compact/tracking/support_service_craterlake.xml

```diff
@@ -0,0 +1,369 @@
+    <constant name="TrackerSupportCylP_thickness2E"        value="TrackerSupportCylAlP_thickness2E + TrackerSupportCylCF_thickness2+ TrackerSupportCylAlP_thickness2SiCone+TrackerSupportCylAlP_thickness2SiDisk" />
+
+    <comment> Negative </comment>
+    <constant name="TrackerSupportCylEndcapN_zmin2A"       value="InnerSupportCone_zmax" /> <comment> ~ 48.75cm </comment>
+    <constant name="TrackerSupportCylEndcapN_zmin2B"       value="105*cm" />
+    <constant name="TrackerSupportCylEndcapN_zmin2C"       value="110*cm" />
+
+    <constant name="TrackerSupportCylEndcapN_zmax2A"       value="TrackerSupportCylEndcapN_zmin2B" />
+    <constant name="TrackerSupportCylEndcapN_zmax2B"       value="TrackerSupportCylEndcapN_zmin2C" />
+    <constant name="TrackerSupportCylEndcapN_zmax2C"       value="InnerTrackerEndcapN_zmax" /> <comment> 123.6cm. Should be 120cm </comment>
+
+    <constant name="TrackerSupportCylEndcapN_z2A"          value="0.5*(TrackerSupportCylEndcapN_zmin2A + TrackerSupportCylEndcapN_zmax2A)" />
+    <constant name="TrackerSupportCylEndcapN_z2B"          value="0.5*(TrackerSupportCylEndcapN_zmin2B + TrackerSupportCylEndcapN_zmax2B)" />
+    <constant name="TrackerSupportCylEndcapN_z2C"          value="0.5*(TrackerSupportCylEndcapN_zmin2C + TrackerSupportCylEndcapN_zmax2C)" />
+
+    <constant name="TrackerSupportCylEndcapN_length2A"     value="TrackerSupportCylEndcapN_zmax2A - TrackerSupportCylEndcapN_zmin2A" />
+    <constant name="TrackerSupportCylEndcapN_length2B"     value="TrackerSupportCylEndcapN_zmax2B - TrackerSupportCylEndcapN_zmin2B" />
+    <constant name="TrackerSupportCylEndcapN_length2C"     value="TrackerSupportCylEndcapN_zmax2C - TrackerSupportCylEndcapN_zmin2C" />
+
+    <comment> cables from Si tracker to outer barrels </comment>
+    <constant name="TrackerSupportCylAlN_thickness2SiCone" value="3.71*mm" />
+    <constant name="TrackerSupportCylAlN_thickness2SiDisk" value="3.04*mm" />
+    <comment> cables from MPGD </comment>
+    <constant name="TrackerSupportCylAlN_thickness2A"      value="1.56*mm" />
+    <constant name="TrackerSupportCylAlN_thickness2B"      value="3.12*mm" />
+    <constant name="TrackerSupportCylAlN_thickness2C"      value="4.43*mm" />
+    <constant name="TrackerSupportCylN_thickness2A"        value="TrackerSupportCylAlN_thickness2A + TrackerSupportCylCF_thickness2+ TrackerSupportCylAlN_thickness2SiCone" />
+    <constant name="TrackerSupportCylN_thickness2B"        value="TrackerSupportCylAlN_thickness2B + TrackerSupportCylCF_thickness2 + TrackerSupportCylAlN_thickness2SiCone+TrackerSupportCylAlN_thickness2SiDisk" />
+    <constant name="TrackerSupportCylN_thickness2C"        value="TrackerSupportCylAlN_thickness2C + TrackerSupportCylCF_thickness2 + TrackerSupportCylAlN_thickness2SiCone+TrackerSupportCylAlN_thickness2SiDisk" />
+
+  </define>
+
+  <detectors>
+    <detector type="epic_SupportServiceMaterial"
+      name="InnerTrackerSupport"
+      id="84"
```

Comment:
Passive materials don't need a detector `id`, though maybe ACTS materials need one(?). In any case, this definitely shouldn't use a hard-coded one here but should use a variable in `compact/definitions.xml`. Now, this overlaps with `TrackerEndcapP_7_ID` which is fortunately not used by any actual detector. This also needs to be fixed in `compact/tracking/central_tracker_hybrid_v2.xml` and `compact/tracking/support_service_assembly.xml`.

### Possible fix 1 (preferred):
Simply remove the `id` entirely and check that this still works (i.e. ACTS doesn't require an `id` for the conversion of passive volumes):
```suggestion
```
Possibly check with @sly2j why he originally hardcoded the `id` field here.

### Possible fix 2:
Use the already existing variable:
```suggestion
      id="TrackerEndcapP_7_ID"
```
but then rename it to something a bit more descriptive. Since we have only a limited number of detector `id` values (up to 256), we don't want to use up values we don't need.

Reply:
I don't  think passive material need an id either but don't have all resources to fully test this hypothesis. For now I simply change it to TrackerSupport_0_ID.